[CK_TILE] Update example README files & fix script compatibility issue (#1548)
ROCm/composable_kernel: example/ck_tile/01_fmha/README.md

```
# in the root of ck_tile
mkdir build && cd build

> bash ../script/cmake-ck-dev.sh  ../ <arch>  # you can replace this <arch> to gfx90a, gfx942...

Reviewer: I'm assuming the architecture needs to be specified. If it doesn't then the wording should be different. ("you can optionally specify the architecture" or something like that)

```suggestion
bash ../script/cmake-ck-dev.sh  ../ <arch>  # replace <arch> with the appropriate architecture, for example gfx90a, gfx942...
```

             -kname    if set to 1 will print kernel name (default:0)
              -init    init method. ui, uniform random int, ni, normalized random int (default:uf)
                       uf, uniform random float, nf, normalized random float, tf, trig float, uf:q, quantization
              -seed    random seed used for initializing input tensors. 0 for non-deterministic seed (default:11939)
            -p_drop    0~1 probability of dropout (default:0)
         -drop_seed    seed for random number generator (default:1)
       -drop_offset    offset for random number generator (default:0)
             -timer    gpu:gpu timer, cpu:cpu timer (default:gpu)
        -rotary_dim    RoPE rotary dimension. rotary_dim <= 0 means not apply RoPE at all (default:0)
-rotary_interleaved    whether to apply interleaved RoPE (default:1)
        -num_splits    # of splits for key/value. 0 to determine actual number by heuristic (default:1)
   -page_block_size    paged-kvcache block size. 0 means not use paged-kvcahe (default:0)
   -cache_batch_idx    whether to use index map to the kvcache (default:0)
            -warmup    number of iterations before benchmark the kernel (default:5)

>             -repeat    number of iterations to benchmark the kernel (default:20)

Reviewer: I realize that this is the output of the output. I suggest that if it's subject to change, to not hardcode it in from a maintenance perspective. I think simply saying "this will provide a list of all the arguments" would be fine (without the "supported").

(Also, the descriptions hardcoded in the code itself need to be fixed, but that's a different issue.)

Author: Thanks. I will leave the existing arguments there and only update the descriptions
